src/timestamp.js

```diff
@@ -0,0 +1,5 @@
+
+export default function convertSecondstoHHMMSS(seconds){    //if the time passed to it is less than an hour, it converts to mm:ss instead
```

Comment:
This function name is kinda long and kinda weird to read. Maybe `secondsToTimestamp`.